app.js

```diff
@@ -22,6 +24,16 @@ const http = require('http-shutdown')(httpBase);
 const io = require('socket.io')(http);
 /* eslint-enable */
 
+// Configuring the i18n
+i18n.use(i18nBackend);
+i18n.init({
+  backend: {
+    loadPath: './www/locales/{{lng}}/{{ns}}.json',
```

Comment:
can we swap the file name and the folder name? make the file `en.json` without the extra folder?